Xamarin/SSW.Consulting/SSW.Consulting/App.xaml.cs

```diff
@@ -74,5 +76,15 @@ private async Task UpdateAccessTokenAsync()
                 }
             }
         }
+
+        private async Task CheckApiCompatibilityAsync()
+        {
+            ApiInfo info = new ApiInfo(Constants.ApiBaseUrl);
+
+            bool compatible = await info.IsApiCompatibleAsync();
+
+            if (!compatible)
+                await Application.Current.MainPage.DisplayAlert("Update Required", "Looks like you're using an older version of the app. You can continue, but some features may not function as expected.", "OK");
```

Comment:
This is a different message from what we have in the PBI.
I do like this one better so maybe we should update the Acceptance Criteria - check with PO

Reply:
Change made as requested